sharepoint/sharepoint-ps/sharepoint-online/Start-SPOAuditDataCollectionForActivityInsights.md

```diff
@@ -0,0 +1,79 @@
+﻿---
+external help file: sharepointonline.xml
+Module Name: Microsoft.Online.SharePoint.PowerShell
+online version: https://learn.microsoft.com/powershell/module/sharepoint-online/start-spoauditdatacollectionforactivityinsights
+applicable: SharePoint Online
+title: Start-SPOAuditDataCollectionForActivityInsights
+schema: 2.0.0
+author: pvrk
+ms.author: pullabhk
+manager: 
+ms.reviewer:
+---
+
+# Start-SPOAuditDataCollectionForActivityInsights
+
+## SYNOPSIS
+
+This cmdlet starts collecting relevant audit data for reports, based on activities within the last 28 days, such as Sharing link reports, content shared with Everyone except external users etc from SharePoint admin center.
+
+## SYNTAX
+
+```
+Start-SPOAuditDataCollectionForActivityInsights -ReportEntity <OptInReportEntityEnum> [<CommonParameters>]
+```
+
+## DESCRIPTION
+
+This cmdlet starts collecting relevant audit data for reports, based on activites related to sharing and access. Reports are available for the following scenarios:
+
+- Sharing links created in last 28 days (Anyone, People-in-your-org, Specific people shared externally).
+- Content shared with Everyone except external users (EEEU) in last 28 days.
+- Copilot agents created in last 28 days *(Private Preview)*
+
+## EXAMPLES
+
+### Example 1
+```powershell
+PS C:\> Start-SPOAuditDataCollectionForActivityInsights -ReportEntity SharingLinks_Anyone
+```
+
+This cmdlet will start collecting audit data related to the generation of 'Anyone' sharing links from the moment it is executed.
```

Comment:
>it [](http://example.com/codeflow?start=113&length=2)

what does "it" refer to here -- the cmdlet or the audit data? Please reword the sentence so its clear

Reply:
apply to the Stop- cmdlet too